.github/workflows/ci.yml

```diff
@@ -41,9 +41,9 @@ jobs:
 
         strategy:
             matrix:
-                tag: [latest]
                 browser: [Chrome, Firefox]
                 python-version: ['3.10']
+                aiida-core-version: [2.1.2, 2.4.3]          # test on the latest and the oldest supported version
```

Comment:
Here version `2.1.2` actually works since it is pre-installed in the image. So we have at least some coverage.